style.css

```diff
@@ -535,3 +545,314 @@ input {
   background: white;
   color: #5e6c84;
 }
+
+@media only screen and (min-width: 992px) {
+  body {
+    background: #e5e5e5;
+    max-width: 100%;
+  }
+
+  .navbar {
+    width: 100%;
+    padding: 0;
+    display: flex;
+    padding-right: 155px;
+    padding-left: 155px;
+    padding-top: 25px;
+    padding-bottom: 25px;
+  }
+
+  .navbar ul,
+  .logo {
+    margin: 0;
+  }
+
+  .navbar ul {
+    display: flex;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navbar ul a {
+    padding-right: 50px;
+    display: flex;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+  }
+
+  .hero {
+    width: 100%;
+    height: 700px;
+    background: rgb(255, 255, 255) url("./images/Header bg.svg") no-repeat;
+    background-size: contain;
+    padding: 24px;
+  }
+
+  .title {
+    margin-top: 80px;
+    margin-bottom: 0;
+  }
+
+  .title h1,
+  .title h2 {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    width: 800px;
+    margin: 0 auto;
+  }
+
+  .description p {
+    width: 800px;
+    height: 98px;
+  }
+
+  .connect {
+    width: 150px;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    margin-top: 0;
+    text-align: left;
+    margin-left: 0;
+  }
+
+  .social {
+    margin-left: 250px;
+    display: block;
+  }
+
+  .social img {
+    width: 20px;
+    height: 20px;
+  }
```

Comment:
- [ ] [OPTIONAL] This section is really well designed 👏. Please try to align the social media icons to the left with the text. So it can match the Figma on the desktop version properly. 
- [ ] Please make sure the mobile version in this section remains working when you make this change
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144680899-6519ebb0-c6ba-4cc9-b0cd-504c670543b4.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/144680955-e1671328-d736-4775-bddb-56f2a6d9fc2c.png)